orval.config.ts

```diff
@@ -16,8 +16,9 @@ export default defineConfig({
       mode: "tags-split",
       mock: {
         type: "msw",
-        // Needs to be shorter than the 1000ms timeout that RTL sets for async methods.
-        delay: Number(env.VITE_MOCK_API_DELAY),
+        delay: () => (process.env.NODE_ENV === "development" ? 1000 : 10),
```

Comment:
Was wondering if maybe something like
```
delay: () => (process.env.NODE_ENV === "test" ? 10 : 1000),
```
might be slightly more flexible, as we are only interested in reducing the time when running the tests? Also, could the time being set to only 10 milliseconds make some rendering timing issues?

Reply:
> Was wondering if maybe something like
> 
> ```
> delay: () => (process.env.NODE_ENV === "test" ? 10 : 1000),
> ```
> 
> might be slightly more flexible, as we are only interested in reducing the time when running the tests?

If we use `NODE_ENV === "test"` then when run on PRs (NODE_ENV === "ci") then we'll get the long timeouts. I think the only scenario where we want the long timeout is when running the demo where `NODE_ENV === "development"`.

> Also, could the time being set to only 10 milliseconds make some rendering timing issues?

If 10ms exposes some rendering timing issues I think that would be a good thing as we'd want to fix those issues. The reason I've gone for such a short time is to make the tests run quickly but also because the time to make all API calls needs to be less than the React Testing Library timeout. In some cases we'll be doing chained calls so we don't want to have to make the timeout too long.